### PR TITLE
feat: project details section

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1651

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie you can test this on https://deploy-preview-1901--regen-registry.netlify.app/project/C02-002

**Comment by erikalogie** [comment]
@blushi this looks great! 

Some small details:
- Let's not have the text stretch so wide on desktop. The increased character count per line makes it difficult to read.
https://monosnap.com/file/hr2Tvg5LOiX3ZzeONuh4vJVPR1B55X
- Add some space on mobile https://monosnap.com/file/yB4t81bKYwRj61T8X99GBOOUOYEofn
- It feels like there is too much space above the module when there is no photo gallery. Is this something that can be fixed easily? We might change the location of this section to further down the page FYI so it may not make sense to spend time on this now. https://monosnap.com/file/mk9r6Nyi6fFWy7nQ9gjPZjyNf2rCi1

**Comment by blushi** [comment]
still need to adjust to add prev/next buttons as discussed with @erikalogie : https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?type=design&node-id=8503-226033&t=oTGvOmBxjgL7chFv-0

**Comment by blushi** [comment]
> * Add some space on mobile https://monosnap.com/file/yB4t81bKYwRj61T8X99GBOOUOYEofn

how much? 

> * It feels like there is too much space above the module when there is no photo gallery. Is this something that can be fixed easily? We might change the location of this section to further down the page FYI so it may not make sense to spend time on this now. https://monosnap.com/file/mk9r6Nyi6fFWy7nQ9gjPZjyNf2rCi1

probably best to fix once we're sure about the sections order

**Comment by erikalogie** [comment]
> > * Add some space on mobile https://monosnap.com/file/yB4t81bKYwRj61T8X99GBOOUOYEofn
> 
> how much?
> 

https://monosnap.com/file/tUyKhomZTxoqHluL96JkvewNJsk8uF

> > * It feels like there is too much space above the module when there is no photo gallery. Is this something that can be fixed easily? We might change the location of this section to further down the page FYI so it may not make sense to spend time on this now. https://monosnap.com/file/mk9r6Nyi6fFWy7nQ9gjPZjyNf2rCi1
> 
> probably best to fix once we're sure about the sections order

ok sounds good

File: web-components/src/components/cards/CredibilityCard/CredibilityCard.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+export const CredibilityCard: React.FC<CredibilityCardProps> = ({
+  index = 0,
+  title,
+  descriptionRaw,
+  icon,
+  claims,
+}) => {
+  const theme = useTheme();
+  const lg = useMediaQuery(theme.breakpoints.up('lg'));
+  const background = `${Math.min(100, 55 + index * (lg ? 12 : 8))}%`;
```
Comment (flagrede) [inline web-components/src/components/cards/CredibilityCard/CredibilityCard.tsx:18]: Could we do the same thing without the client media query check? This is to start making components that are SSR-friendly.

Something like that:
```
const backgroundMobile = `${Math.min(100, 55 + index * 8)}%`;
const backgroundDesktop = `${Math.min(100, 55 + index * 12)}%`;

background: {xs: `linear-gradient(-8deg, #F1F7F6 ${backgroundMobile}, transparent ${backgroundMobile})`, lg: `linear-gradient(-8deg, #F1F7F6 ${backgroundDesktop}, transparent ${backgroundDesktop})`}
```

File: web-components/src/components/sliders/ResponsiveSlider.tsx [hunk 2]
```diff
@@ -221,7 +229,7 @@ export default function ResponsiveSlider({
       <Grid
         container
         wrap="nowrap"
-        alignItems="center"
+        alignItems="flex-end"
```
Comment (flagrede) [inline web-components/src/components/sliders/ResponsiveSlider.tsx:232]: Could this change create visual regression with other used sliders or do you think it's safe to do?

File: web-components/src/components/sliders/ResponsiveSlider.tsx [hunk 3]
```diff
@@ -238,8 +246,7 @@ export default function ResponsiveSlider({
         ) : null}
         {items.length > 1 && arrows && desktop && (
           <Grid
-            xs={12}
-            sm={4}
+            xs={renderTitle || title ? 4 : 12}
```
Comment (flagrede) [inline web-components/src/components/sliders/ResponsiveSlider.tsx:249]: I'm confused about why we sent `renderTitle`, a function of type `() => JSX.Element` to this prop that seems to accept `number | bool` ?

File: web-registry/src/components/organisms/ProjectDetailsSection/ProjectDetailsSection.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+import { useSectionStyles } from './ProjectDetailsSection.styles';
+import { ProjectDetailsSectionProps } from './ProjectDetailsSection.types';
+
+export const ProjectDetailsSection: React.FC<ProjectDetailsSectionProps> = ({
+  header,
+  credibilityCards,
+}) => {
+  const theme = useTheme<Theme>();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```
Comment (flagrede) [inline web-registry/src/components/organisms/ProjectDetailsSection/ProjectDetailsSection.tsx:18]: Same comment as above, would be nice to have that without client MQ check to be more SSR friendly.

**Review by flagrede — COMMENTED** [review]
Nice work, the responsive looks particularly tricky for this one!
Left a few comments/questions.

File: web-components/src/components/cards/CredibilityCard/CredibilityCard.tsx (hunk 1 above)
Comment (blushi) [reply web-components/src/components/cards/CredibilityCard/CredibilityCard.tsx]: good point, I added some util fn to make the code more readable

File: web-components/src/components/sliders/ResponsiveSlider.tsx (hunk 3 above)
Comment (blushi) [reply web-components/src/components/sliders/ResponsiveSlider.tsx:249]: we set `xs` to 4 or 12 in this case

File: web-components/src/components/sliders/ResponsiveSlider.tsx (hunk 2 above)
Comment (blushi) [reply web-components/src/components/sliders/ResponsiveSlider.tsx:232]: will set this as a prop instead to be sure

File: web-components/src/components/sliders/ResponsiveSlider.tsx (hunk 3 above)
Comment (flagrede) [reply web-components/src/components/sliders/ResponsiveSlider.tsx:249]: I thought the `||` operator would take priority over the ternary but apparently, that's not the case, all good then.

File: web-registry/src/components/organisms/ProjectDetailsSection/ProjectDetailsSection.tsx (hunk 4 above)
Comment (flagrede) [reply web-registry/src/components/organisms/ProjectDetailsSection/ProjectDetailsSection.tsx:18]: I understand, let's keep it like that then.

**Comment by erikalogie** [comment]
@blushi would it be possible for you to add another 2 cards so I can better get a feel for the back and next arrow functionality? https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?type=design&node-id=6959%3A184592&t=nD5YlNpGJNhieHfK-1 i.e. additional and permanent here.

**Comment by blushi** [comment]
> @blushi would it be possible for you to add another 2 cards so I can better get a feel for the back and next arrow functionality? https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?type=design&node-id=6959%3A184592&t=nD5YlNpGJNhieHfK-1 i.e. additional and permanent here.

added

**Pull request merged**